couchdb/source/pkg/adapter/adapter.go

```diff
@@ -49,10 +53,21 @@ type couchDbAdapter struct {
 	logger    *zap.SugaredLogger
 
 	source  string
+	feed    string
 	couchDB *kivik.DB
 	options kivik.Options
 }
 
+func init() {
+	var transport = http2.Transport{
+		DisableCompression: true,
+	}
+	kivik.Register("cloudant", &couchdb.Couch{
```

Comment:
seeing `cloudant` here, would this mean it does not work w/ "vanilla" Apache CouchDB server?